src/coreclr/vm/array.cpp

```diff
@@ -402,8 +402,7 @@ MethodTable* Module::CreateArrayMethodTable(TypeHandle elemTypeHnd, CorElementTy
             StackSString ssElemName;
             elemTypeHnd.GetName(ssElemName);
 
-            StackScratchBuffer scratch;
```

Comment:
Is the `GetUTF8` method that takes the scratch buffer still needed after this cleanup?

Reply:
There's a few places I didn't remove it due to possible perf concerns, but we can move all of those cases to use `ConvertToUTF8` + `GetUTF8` on the stack copy. I'll look into doing that.